Exiled.Events/Commands/Reload/All.cs

```diff
@@ -0,0 +1,60 @@
+// -----------------------------------------------------------------------
+// <copyright file="All.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Commands.Reload
+{
+    using System;
+
+    using CommandSystem;
+
+    using Exiled.Loader;
+    using Exiled.Permissions.Extensions;
+
+    /// <summary>
+    /// The reload configs command.
+    /// </summary>
+    public class All : ICommand
+    {
+        /// <inheritdoc/>
+        public string Command { get; } = "all";
+
+        /// <inheritdoc/>
+        public string[] Aliases { get; } = new string[] { "a" };
+
+        /// <inheritdoc/>
+        public string Description { get; } = "Reload all configs and plugins.";
+
+        /// <inheritdoc/>
+        public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out string response)
+        {
+            bool success = true;
+
+            if (!new Configs().Execute(arguments, sender, out string responsetemp))
```

Comment:
Don't create a new instance for one call, make a static instance variable/property in command classes and also use the variable/property to register them in a parent command

Reply:
Done!

Reply:
Thank you!